controllers/main.js

```diff
@@ -1,9 +1,9 @@
 const { getPixelatedImage } = require("../middleware");
 
-const style = require("../public/javascripts/extraStyles.js");
+const { cardStyle } = require("../public/javascripts/extraStyles.js");
 
 module.exports.renderPlay = async (req, res, next) => {
   const image = await getPixelatedImage(req.gameData.background_image);
-  const extraStyles = style.cardStyle;
+  const extraStyles = cardStyle;
```

Comment:
Ok this looks better already, but I what I meant is that you don't need to declare a constant with the name `extraStyles` to be able to pass it below. 

Take a look at this: 

```res.render("main/play.ejs", { image, extraStyles: cardStyle });```

Can safely remove the `extraStyles` constant you declare in the route!